widgets/htdocs/alphafold/controlPanel.js

```diff
@@ -0,0 +1,38 @@
+import { html, css, LitElement } from 'https://unpkg.com/lit@2.0.0-rc.4/index.js?module';
```

Comment:
i dont understand whats happening here?

Reply:
thinking of this, what if this module is not available from that cdn or if the cdn is down?
Do you think having this library locally would be good?

Reply:
> i dont understand whats happening here?

A third-party library is imported.

> thinking of this, what if this module is not available from that cdn or if the cdn is down? Do you think having this library locally would be good?

Unkpg is powered by Cloudflare (see [their home page](https://unpkg.com)); should be fairly robust.

Reply:
sorry i meant the whole file, what does it do?

Reply:
It's encodes the visual appearance of a panel as a common component. A block with a title and a body. The styles for the title are the same across all such panels (exons, protein features, variants). What's different is what goes into the body; and that is defined in each specific panel file (exonsControlPanel, variantsControlPanel, etc.)

<img src="https://user-images.githubusercontent.com/6834224/134591382-9f585410-67e9-414f-90c5-b97fc2b2c184.png" width="400" />